### DISCUSSION TITLE
Numbered List to continue after interruption

### DISCUSSION THREAD

**Discussion by ZarTrox** [Azure/api-management-developer-portal#2566] [Feature request]
## Bug description

When working with numbered lists it is sometimes necessary to interrupt it with e.g. an image. So it would be beneficial if there could be an option to let it continue after this interruption in another text block. (Or to be able to insert an element into an other element - here: an image into a text segment)

## Is your portal managed or self-hosted?

Managed

## Additional context

Currently:
<img width="808" alt="Screenshot 2021-01-12 101642" src="https://user-images.githubusercontent.com/58420097/104294341-56ff7100-54bf-11eb-97c7-3e96d8835e95.png">

Ideally:
<img width="808" alt="Screenshot 2021-01-12 101642" src="https://user-images.githubusercontent.com/58420097/104294722-d68d4000-54bf-11eb-8bc9-d53130649a28.png">

**Comment by AnRei123** [comment]
**Intended Behavior:** 
Configure Text-widget so that a numbered list will be continued according to the last numbered list of the previous text widget irrespective if it is in a different or in the same block/section.

**Comment by ghost** [comment]
@ZarTrox, thank you for requesting this feature.
